spec/models/metric/ci_mixin/capture_spec.rb

```diff
@@ -186,6 +187,15 @@ def mock_adapter(postgre_adapter)
             end
           end
         end
+
+        context "with a paused EMS" do
+          let(:ems_openstack) { FactoryBot.create(:ems_openstack, :zone => Zone.maintenance_zone, :enabled => false) }
```

Comment:
Wonder if this can be extracted to a trait on the ext_management_system.  e.g. `:paused` or `:disabled`.  Not for this PR.

Reply:
Yeah I was going to do that but the `Zone.seed` that was needed to have `Zone.maintenance_zone` complicated that a bit

Reply:
Yeah I figured.

Reply:
I too looked into this. aborting/avoiding the ✨
I think a before that requires `Zone.seed` is reasonable if you want to go that route.

this is good though.
keep / :shipit: